app/views/posts/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<a href="<%= users_path() %>">
+  <button class="custom-back-btn">⬅</button>
+</a>
+<ul>
+  <a href="<%= user_path(@user) %>">
```

Comment:
- [ ] Please fix the path here as well by replacing this code `<a href="<%= user_path(@user) %>">` with this ` <a href="<%= user_posts_path() %>">`

Reply:
This change has been done well